ruby-3.2-recursive-open-struct.yaml

```
@@ -0,0 +1,48 @@
+package:
+  name: ruby-3.2-recursive-open-struct
+  version: 1.2.2
+  epoch: 0
+  description: OpenStruct subclass that returns nested hash attributes as RecursiveOpenStructs
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - ruby-3.2
+      - ruby-3.2-dev
+
+vars:
+  gem: recursive-open-struct
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 3bb03300317fa0e5303e53f82881f6f3c74f6bad
+      repository: https://github.com/aetherknight/recursive-open-struct
+      tag: v${{package.version}}
+
+  - uses: ruby/build
+    with:
+      gem: ${{vars.gem}}
+
+  - uses: ruby/install
+    with:
+      gem: ${{vars.gem}}
+      version: ${{package.version}}
+
+  - uses: ruby/clean
+
+test:
+  pipeline:
+    - runs: |
+        ruby -e "require 'recursive-open-struct'"
+
```

Can we add something like:
https://github.com/aetherknight/recursive-open-struct?tab=readme-ov-file#usage

```suggestion
     - name: Simple test
       runs: |
         cat <<EOF > /tmp/test.rb
         require 'recursive-open-struct'
         ros = RecursiveOpenStruct.new( { wha: { tagoo: 'siam' } } )
         ros.wha.tagoo
         EOF
         ruby /tmp/test.rb
```